bower.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "origin-web-comon",
+  "version": "0.0.1",
+  "main": [
+    "dist/origin-web-common.js"
+  ],
+  "authors": [
+    "Red Hat"
+  ],
+  "description": "Library of common services and components for OpenShift Web Console.",
+  "license": "Apache-2.0",
+  "ignore": [
+    "Gruntfile.js",
+    "package.json",
+    ".bowerrc",
+    ".editorconfig",
+    ".gitignore",
+    ".jshintrc",
+    "src",
+    "README.md",
+    "bower.json",
+    ".bower.json",
+    "grunt-ngdocs-index.tmpl"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/openshift/origin-web-common.git"
+  },
+  "dependencies": {
+    "angular": "1.5.11",
```

Comment:
Wondering if we shouldn't allow a range for these dependencies to make them easier to update downstream. @jwforres opinion?